synthesizer/grid.py

```diff
@@ -245,7 +245,7 @@ def get_spectra(self):
 
                 # if incident is one of the spectra also add a spectra called "stellar"
                 if spec_name == 'incident':
-                    self.spectra['stellar'] = self.spectra[spec_name]
+                    self.spectra["stellar"] = self.spectra[spec_name]
```

Comment:
if this is going to be turned into a more general grid class, we should be careful to update this